packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const shortid = require('shortid');
```

Comment:
I still hesitate doing random key generation, even in examples, because we know this is a pattern we don't want to perpetuate. Can we make the examples slightly more meaningful, allowing us to make use keys that'd make sense? Something like:

```
<Table isStriped={false}>
  <Table.Header>
    <Table.HeaderContent content={'Name'} key="NAME" minWidth={'small'} />
    <Table.HeaderContent content={'Address'} key="ADDRESS" minWidth={'medium'} />
    <Table.HeaderContent content={'Phone'} key="PHONE_NUMBER" minWidth={'large'} />
  </Table.Header>
  <Table.Rows>
    <Table.Row key="PERSON_0">
      <Table.RowContent content="John Smith" key="NAME" />
      <Table.RowContent content="123 Adams Drive" key="ADDRESS" />
      <Table.RowContent content="111-222-3333" key="PHONE_NUMBER" />
    </Table.Row>
    <Table.Row key="PERSON_1">
      <Table.RowContent content="Jane Smith" key="NAME" />
      <Table.RowContent content="321 Drive Street" key="ADDRESS" />
      <Table.RowContent content="111-222-3333" key="PHONE_NUMBER" />
    </Table.Row>
    <Table.Row key="PERSON_2">
      <Table.RowContent content="Dave Smith" key="NAME" />
      <Table.RowContent content="213 Raymond Road" key="ADDRESS" />
      <Table.RowContent content="111-222-3333" key="PHONE_NUMBER" />
    </Table.Row>
  </Table.Rows>
</Table>
```

We want to get the point across that the key structure should match the schema of whatever data structure is being presented.